simapp/test_helpers.go

```diff
@@ -361,10 +363,12 @@ func SignCheckDeliver(
 // GenSequenceOfTxs generates a set of signed transactions of messages, such
 // that they differ only by having the sequence numbers incremented between
 // every transaction.
-func GenSequenceOfTxs(msgs []sdk.Msg, accNums []uint64, initSeqNums []uint64, numToGenerate int, priv ...crypto.PrivKey) []authtypes.StdTx {
-	txs := make([]authtypes.StdTx, numToGenerate)
+func GenSequenceOfTxs(msgs []sdk.Msg, txGen client.TxGenerator, accNums []uint64, initSeqNums []uint64, numToGenerate int, priv ...crypto.PrivKey) ([]sdk.Tx, error) {
```

Comment:
Seems `txGen` should be the first param, no?
```suggestion
func GenSequenceOfTxs( txGen client.TxGenerator, msgs []sdk.Msg, accNums []uint64, initSeqNums []uint64, numToGenerate int, priv ...crypto.PrivKey) ([]sdk.Tx, error) {
```